docs/advanced-features/compiler.md

```diff
@@ -312,13 +319,36 @@ module.exports = {
 }
 ```
 
-Advanced transformations:
+#### Handlebars variables and helper functions
+
+This transform uses [handlebars](https://docs.rs/handlebars) to template the replacement import path in the `transform` field. These variables and helper functions are available:
+
+1. `member`: Has type `string`. The name of the member import.
+2. `lowerCase`, `upperCase`, `camelCase`, `kebabCase`: Helper functions to convert a string to lower, upper, camel or kebab cases.
+3. `matches`: Has type `string[]`. All groups matched by the regular expression. `matches.[0]` is the full match.
```

Comment:
I reordered these. Previously it was:

> 1. `matches`: ...
> 2. `member`: ...
> 3. `lowerCase`, `upperCase`, `camelCase`, `kebabCase`: ...

But you'd likely use use `member` the most, then case helpers. So I re-ordered.